### ISSUE TITLE
Initial window should run example system

### ISSUE THREAD

**Issue by andeplane** [ComputationalPhysics/atomify-lammps#192]
When user starts Atomify for the first time, it should already be running an example. This way, the app feels much more interactive from the very beginning and the user can more quickly start.

**Comment by andeplane** [comment]
Currently, it works on my computer with
```
QString initialScript("/Users/anderhaf/Desktop/old/lj.in");
    QFileInfo info(initialScript);
    if(info.exists()) {
        m_scriptFilePath = initialScript;
    }
```
in constructor of AtomifySimulator.  MainDesktop.qml has
```
Timer {
        id: startsimtimer
        interval: 250
        onTriggered: {
            if(visualizer.simulator.scriptFilePath !== "") {
                visualizer.simulator.started()
            }
        }
    }

    Component.onCompleted: {
        startsimtimer.start()
    }
```

which starts the simulation. This is a bit hacky, but works. We should at least change the path to one of the example systems so it's general on all computers.

**Comment by andeplane** [comment]
Fixed in ec471bf
